libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -253,8 +253,8 @@ class AC_PosControl
     /// set_pos_target_xy_cm - sets the position target, frame NEU in cm relative to the EKF origin
     void set_pos_target_xy_cm(float pos_x, float pos_y) { _pos_target.x = pos_x; _pos_target.y = pos_y; }
 
-    /// get_pos_target_cm - returns the position target, frame NEU in cm relative to the EKF origin
-    const Vector3p& get_pos_target_cm() const { return _pos_target; }
+    /// get_pos_target_cm - returns the position target (not including offsets), frame NEU in cm relative to the EKF origin
+    Vector3p get_pos_target_cm() const { return _pos_target - _pos_offset; }
```

Comment:
this changes from a reference to a returned value, we need to change the use cases in places like mode_guided, need to change this:
```
        guided_pos_target_cm.x = pos_control->get_pos_target_cm().x;
        guided_pos_target_cm.y = pos_control->get_pos_target_cm().y;
```
to 
```
guided_pos_target_cm.xy() = pos_control->get_pos_target_cm().xy();
```

Reply:
should also consider moving to the cpp from the .h

Reply:
I've done both these suggestions now, thanks